opp_gui/src/widgets/tool_path_planner_widget.cpp

```diff
@@ -303,7 +305,7 @@ void ToolPathPlannerWidget::saveModel()
   using TouchPointMap = std::map<std::string, opp_msgs::TouchPoint>;
   TouchPointMap touch_points = touch_point_editor_->getPoints();
   TouchPointMap verification_points = verification_point_editor_->getPoints();
-  if (touch_points.size() < 3 || verification_points.size() < 3)
+  if (touch_points.size() < MIN_TOUCH_POINTS || verification_points.size() < MIN_VERIFICATION_POINTS)
   {
     QMessageBox::warning(
         this, "Invalid Model Definition", "Ensure at least 3 touch points and 3 verification points have been defined");
```

Comment:
I would also change this message to indicate the number of touch and verification points that are required